src/main/java/race/Car.java

```diff
@@ -6,14 +6,44 @@
 
 public class Car {
 
-    private List<Boolean> distance = new ArrayList<>();
 
-    public List<Boolean> getDistance() {
-        return Collections.unmodifiableList(distance);
+    private String name;
+
+    private List<Boolean> winOrLoseLog = new ArrayList<>();
+
+    private int distance;
+
+    public List<Boolean> getWinOrLoseLog() {
+        return Collections.unmodifiableList(winOrLoseLog);
+    }
+
+    public String getName() {
+        return name;
     }
 
     public void addDistance(MoveCondition moveCondition) {
-        distance.add(moveCondition.moveOrNot());
+        boolean result = moveCondition.moveOrNot();
+        winOrLoseLog.add(result);
+        setDistance(result);
+    }
+
+    private void setDistance(boolean result) {
+        if (result)
+            this.distance += 1;
     }
 
+    public int getDistance() {
+        return distance;
+    }
+
+    public static Car createCar(String name, List<Boolean> distance) {
+        Car car = new Car();
+        car.name = name;
+        car.winOrLoseLog = distance;
+        return car;
+    }
+
+
+    public Car() {
+    }
```

Comment:
생성자가 public인데 생성시점에 멤버변수를 초기화하고 있지 않아요! 
그래서 이렇게 표현해주면 어떨까요?

```suggestion
    private Car(String name, List<Boolean> distance) {
      this.name = name;
      this.winOrLoseLog = distance;
    }
    
    public static Car createCar(String name, List<Boolean> distance) {
        return new Car(name, distance);
    }
```

그리고 생성자는 클래스 최상단 (멤버변수 다음)에 위치해주세요!